x-pack/plugins/apm/public/components/app/mobile/errors_and_crashes_overview/crash_group_list/index.tsx

```diff
@@ -66,7 +80,45 @@ function MobileCrashGroupList({
   const columns = useMemo(() => {
     return [
       {
-        name: i18n.translate('xpack.apm.errorsTable.typeColumnLabel', {
+        name: (
+          <>
+            {i18n.translate('xpack.apm.crashTable.groupIdColumnLabel', {
+              defaultMessage: 'Group ID',
+            })}{' '}
+            <EuiIconTip
+              size="s"
+              type="questionInCircle"
+              color="subdued"
+              iconProps={{
+                className: 'eui-alignTop',
+              }}
+              content={i18n.translate(
+                'xpack.apm.crashTable.groupIdColumnDescription',
+                {
+                  defaultMessage:
+                    'Hash of the stack trace. Groups similar errors together, even when the error message is different due to dynamic parameters.',
+                }
+              )}
+            />
+          </>
+        ),
+        field: 'groupId',
+        sortable: false,
+        width: `${unit * 6}px`,
+        render: (_, item: ErrorGroupItem) => {
```

Comment:
I think if you type the columns like below, you don't need to manually type the `item` here. Just a suggestion.

```
Array<ITableColumn<ErrorGroupItem>> 
```

Reply:
On that note: isn't there types in EUI we can use instead of `ITableColumn`?

Reply:
probably this?
```
Array<EuiBasicTableColumn<T>>
```

Reply:
Thanks for the suggestions. I fixed the type column. 

As for the custom one, it requires to update all the components that use the `managed_table`

https://github.com/elastic/kibana/blob/5052a35160a306bf72b08f179715121dfab86379/x-pack/plugins/apm/public/components/shared/managed_table/index.tsx#L16

I would rather do it separately from this PR

Reply:
I think you can leave the `ManagedTable` out of it.